drivers/lcd_common/Makefile.dep

```diff
@@ -0,0 +1,11 @@
+FEATURES_REQUIRED += periph_spi
+FEATURES_REQUIRED += periph_gpio
+USEMODULE += xtimer
+
+DEFAULT_MODULE += ili9341
+ifneq (,$(filter ili9341,$(USEMODULE)))
+  DISABLE_MODULE += st7735
+endif
+ifneq (,$(filter st7735,$(USEMODULE)))
+  DISABLE_MODULE += ili9341
+endif
```

Comment:
I don't really understand the need for a DEFAULT_MODULE here. A simple direct dependency seems enough: if the user wants to use ili9341, then lcd_common is pulled in automatically.

Maybe it's because the current approach prevents one ili9341 and one st7735 display module to be used at the same time ?